coverage_tables.md

```diff
@@ -0,0 +1,387 @@
+| ones | ✅ |
+| ones_like | ✅ |
+| zeros | ✅ |
+| zeros_like | ✅ |
+| full | ✅ |
+| full_like | ✅ |
+| array | ✅ |
+| asarray | ✅ |
+| asanyarray | ❌ |
+| ascontiguousarray | ❌ |
+| asmatrix | ❌ |
+| copy | ✅ |
+| frombuffer | ❌ |
+| from_dlpack | ❌ |
+| fromfile | ❌ |
+| fromfunction | ❌ |
+| fromiter | ❌ |
+| fromstring | ❌ |
+| loadtxt | ❌ |
+| arange | ✅ |
+| linspace | ✅ |
+| logspace | ✅ |
+| geomspace | ❌ |
+| meshgrid | ✅ |
+| mgrid | ❌ |
+| ogrid | ❌ |
+| diag | ✅ |
+| diagflat | ❌ |
+| tri | ❌ |
+| tril | ✅ |
+| triu | ✅ |
+| vander | ❌ |
+| mat | ❌ |
+| bmat | ❌ |
+## NumPy Array Manipulation
+[Back to Table of Contents](#table-of-contents)
+
+| NumPy Array Manipulation | Heat |
+|---|---|
+| copyto | ❌ |
+| shape | ✅ |
+| reshape | ✅ |
+| ravel | ✅ |
+| flat | ❌ |
+| flatten | ✅ |
+| moveaxis | ✅ |
+| rollaxis | ❌ |
+| swapaxes | ✅ |
+| T | ❌ |
+| transpose | ✅ |
+| atleast_1d | ❌ |
+| atleast_2d | ❌ |
+| atleast_3d | ❌ |
+| broadcast | ❌ |
+| broadcast_to | ✅ |
+| broadcast_arrays | ✅ |
+| expand_dims | ✅ |
+| squeeze | ✅ |
+| asarray | ✅ |
+| asanyarray | ❌ |
+| asmatrix | ❌ |
+| asfarray | ❌ |
+| asfortranarray | ❌ |
+| ascontiguousarray | ❌ |
+| asarray_chkfinite | ❌ |
+| require | ❌ |
+| concatenate | ✅ |
+| stack | ✅ |
+| block | ❌ |
+| vstack | ✅ |
+| hstack | ✅ |
+| dstack | ❌ |
+| column_stack | ✅ |
+| row_stack | ✅ |
+| split | ✅ |
+| array_split | ❌ |
+| dsplit | ✅ |
+| hsplit | ✅ |
+| vsplit | ✅ |
+| tile | ✅ |
+| repeat | ✅ |
+| delete | ❌ |
+| insert | ❌ |
+| append | ❌ |
+| resize | ❌ |
+| trim_zeros | ❌ |
+| unique | ✅ |
+| flip | ✅ |
+| fliplr | ✅ |
+| flipud | ✅ |
+| reshape | ✅ |
+| roll | ✅ |
+| rot90 | ✅ |
+## NumPy Binary Operations
+[Back to Table of Contents](#table-of-contents)
+
+| NumPy Binary Operations | Heat |
+|---|---|
+| bitwise_and | ✅ |
+| bitwise_or | ✅ |
+| bitwise_xor | ✅ |
+| invert | ✅ |
+| left_shift | ✅ |
+| right_shift | ✅ |
+| packbits | ❌ |
+| unpackbits | ❌ |
+| binary_repr | ❌ |
+## NumPy IO Operations
+[Back to Table of Contents](#table-of-contents)
+
+| NumPy IO Operations | Heat |
+|---|---|
+| load | ✅ |
+| save | ✅ |
+| savez | ❌ |
+| savez_compressed | ❌ |
+| loadtxt | ❌ |
+| savetxt | ❌ |
+| genfromtxt | ❌ |
+| fromregex | ❌ |
+| fromstring | ❌ |
+| tofile | ❌ |
+| tolist | ❌ |
+| array2string | ❌ |
+| array_repr | ❌ |
+| array_str | ❌ |
+| format_float_positional | ❌ |
+| format_float_scientific | ❌ |
+| memmap | ❌ |
+| open_memmap | ❌ |
+| set_printoptions | ✅ |
+| get_printoptions | ✅ |
+| set_string_function | ❌ |
+| printoptions | ❌ |
+| binary_repr | ❌ |
+| base_repr | ❌ |
+| DataSource | ❌ |
+| format | ❌ |
+## NumPy LinAlg Operations
+[Back to Table of Contents](#table-of-contents)
+
+| NumPy LinAlg Operations | Heat |
+|---|---|
+| dot | ✅ |
+| linalg.multi_dot | ❌ |
+| vdot | ✅ |
+| inner | ❌ |
+| outer | ✅ |
+| matmul | ✅ |
+| tensordot | ❌ |
+| einsum | ❌ |
+| einsum_path | ❌ |
+| linalg.matrix_power | ❌ |
+| kron | ❌ |
+| linalg.cholesky | ❌ |
+| linalg.qr | ❌ |
```

Comment:
here, I'm sure that we have `qr`. Do we need to check the entire list for correctness or is there a reason why this has not been recognized? 

(I'm also quite sure that we have `norm`...)

Reply:
If required, I can do the check - just assign me.

Reply:
@mrfh92 the mismatch arises in `scripts/numpy_coverage_tables.py`, if you have time to adapt it (including path and name of output file, i think that's outdated), it would be awesome. 

We also need to expand the script to some scipy and scikit-learn modules, but we can do it in increments.

Reply:
Done. I have also added the random-module of NumPy.